xml/hwbp_registers.xml

```diff
@@ -270,23 +285,86 @@
             by 1.
         </field>
 
-        <field name="action" bits="5:0" access="R/W" reset="0">
-            Determines what happens when this trigger matches.
-
-            0: Raise a breakpoint exception. (Used when software wants to use the
-            trigger module without an external debugger attached.)
+        <field name="action" bits="5:0" access="R/W" reset="0" />
+    </register>
 
-            1: Enter Debug Mode. (Only supported when \Fdmode is 1.)
+    <register name="Exception Trigger" short="etrigger" address="0x7a1">
+        This register is accessible as \Rtdataone when \Ftype is 4.
 
-            2: Start tracing.
+        If implemented, it allows triggering on a variety of interrupt and
+        exception sources. If such a trigger is taken, all the usual CSRs as
+        well as \Rpc are updated, and the requested action is taken just before
+        the first instruction of the interrupt/exception handler is executed.
 
-            3: Stop tracing.
+        The exceptions to trigger on are specified in \Rtdatatwo. Each of the
```

Comment:
This  is still very brittle. The ISA manual allows users to define higher-bit interrupts all the way to XLEN. I think we should just define \Rtdatatwo to be the full XLEN width, and not try to share it like this between exceptions and interrupts.